aspnetcore/security/authentication/configure-jwt-bearer-authentication.md

```diff
@@ -0,0 +1,323 @@
+---
+title: Configure JWT bearer authentication in ASP.NET Core
+author: damienbod
+description: Learn how to set up JWT bearer authentication in an ASP.NET Core app.
+monikerRange: '>= aspnetcore-8.0'
+ms.author: riande
+ms.custom: mvc
+ms.date: 12/7/2024
+uid: security/authentication/configure-jwt-bearer-authentication
+---
+# Configure JWT bearer authentication in ASP.NET Core
+
+By [Damien Bowden](https://github.com/damienbod)
+
+JWT (JSON Web Token) Bearer Authentication is commonly utilized for APIs. While it operates similarly to cookie authentication, the identity provider issues a JWT or tokens upon a successful authentication which can then be sent to other servers to authenticate unlike cookies which are only sent back to the issuing domain. A JWT is a self-contained token that encapsulates information for an API resource or a client. The client which requested the JWT can request data from an API resource using the Authorization header and a bearer token.
+
```

Comment:
```suggestion

For an introduction to JWT Bearer Authentication, see [JSON Web Tokens](https://jwt.io/introduction/.)
```

Reply:
```suggestion

For an introduction to JWT Bearer Authentication, see [JSON Web Tokens.](https://en.wikipedia.org/wiki/JSON_Web_Token)
```

It's pretty similar content, but Wikipedia is more in-depth vender neutral. I also think people new to the space will more likely be overwhelmed by mentions of HMAC, RSA and ECDSA without links in the Okta intro than the intro to the Wikipedia article which gives a good example of a "logged in as administrator" claim in the intro. The Okta only explains claims later on in the "Payload" section.